src/test/java/org/jenkinsci/plugins/workflow/testMetaStep/DisplaynameEscapeCharData.java

```diff
@@ -0,0 +1,24 @@
+package org.jenkinsci.plugins.workflow.testMetaStep;
+
+import hudson.Extension;
+import hudson.model.AbstractDescribableImpl;
+import hudson.model.Descriptor;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+public class DisplaynameEscapeCharData extends AbstractDescribableImpl<DisplaynameEscapeCharData> {
```

Comment:
Removed unnecessary stuff, the tests now has just a step with no args and no-op execution.
Removed the DisplaynameEscapeCharData.java file as well.